chpl/chpl-api/e2e/script/newman-run.js

```diff
@@ -32,12 +32,25 @@ const apiKeyControllerTests = {
     },
 };
 
-const certifiedProductControllerTests = {
+const certifiedProductControllerDataTests = {
   ...commonOptions,
   collection: collection_path + '/certified-product-controller.postman_collection.json',
+  folder: 'GET : certified_product: data driven tests',
+  iterationData: data_path + '/chplids-test-data.json',
   reporter: {
     junit: {
-      export: reports_path + '/certified-product-controller-tests.xml',
+      export: reports_path + '/certified-product-controller-data-tests.xml',
+    },
+  },
+};
+
+const certifiedProductControllerNoDataTests = {
+  ...commonOptions,
+  collection: collection_path + '/certified-product-controller.postman_collection.json',
```

Comment:
How can the same json file run both the no-data and the data tests?

Reply:
This has been discussed and no changes needed.